portal/settings.py

```diff
@@ -25,7 +25,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['127.0.0.1']
```

Comment:
It's not time yet, to decide what all should be in allowed_hosts. Infact, I hope you understand why that setting exists. Perhaps, we will be creating `server_settings.py` and `local_settings.py` just like cms once we are ready for deployment.

Reply:
okay